File: cs/h00.override/Triangle.cs
public class Triangle : Shape {
  /******* CONSTRUCTORS *******/
  public Triangle(double width, double height) : base(width, height) { /* nothing */ }
  public Triangle(int width, int height) : base(width, height) { /* nothing */ }

  /******* METHODS *******/
  public override double calculateArea() {
    return (base.width * base.height)/2;
  }
}

File: cpp/h00.override/Triangle.cpp
#include "Triangle.h"

namespace TriangleSpace {

 /******* CONSTRUCTORS *******/
 Triangle::Triangle (double w, double h) : Shape(w,h) { /* nothing */ }

 /******* METHODS *******/
 double Triangle::calculateArea () { return (width * height / 2); }

}
